lib/utils/screenshots.js

```diff
@@ -33,21 +33,24 @@ class Screenshots {
    * @param {function} cb
    */
   static writeScreenshotToFile(fileName, content, cb = function() {}) {
-    const dir = path.resolve(fileName, '..');
-
-    mkpath(dir, function(err) {
-      if (err) {
-        cb(err);
-      } else {
-        fs.writeFile(fileName, content, 'base64', function(err) {
-          if (err) {
-            cb(err);
-          } else {
-            cb(null, fileName);
-          }
-        });
-      }
-    });
+    try {
+      const dir = path.resolve(fileName, '..');
+      mkpath(dir, function(err) {
+        if (err) {
+          cb(err);
+        } else {
+          fs.writeFile(fileName, content, 'base64', function(err) {
+            if (err) {
+              cb(err);
+            } else {
+              cb(null, fileName);
+            }
+          });
+        }
+      });
+    } catch (err) {
+      cb(err);
+    } 
```

Comment:
IMO, `try/catch` would not be useful here, as both `mkpath` and `fs.writeFile` is passing error in their callbacks. Hence, `try` would never fail. Any specific reason to use `try/catch` here?

Reply:
You are correct, I was not sure where errors were coming from hence I added it at one point. Not required.